functions/gcs/main_test.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018, Google, LLC.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import main
+
+
+class Context(object):
```

Comment:
Can we import unittest.mock's Mock object and use it here instead of creating this class? (I've updated the other Functions tests to do that as well)